UI/GameSettingsScreen.cpp

```diff
@@ -24,6 +24,7 @@
 #include "math/curves.h"
 #include "gfx_es2/gpu_features.h"
 #include "gfx_es2/draw_buffer.h"
+#include "gfx/d3d9_state.h"
```

Comment:
Including this here breaks the build on non-windows platforms :) Need some #ifdef .